README.md

```diff
@@ -0,0 +1,19 @@
+# PokemonTCGApp
+
+Hello, my name is Marcos Cifuentes, and I'm currently a trainee at Globant in Mobile Studio.
+
+As part of my training, my task was to create an app based on the [POKEMON TCG API](https://docs.pokemontcg.io/#documentationgetting_started), that returns information abut Pokemon cards.
+
+For the development of the app was used [Kotlin language](https://kotlinlang.org/) and [MVVM architecture](https://docs.google.com/presentation/d/1aBtccGraTyggnIP6Nn7m8uGfBgreKWIk-2JuLafKAds/edit#slide=id.p4) together with [Clean](https://proandroiddev.com/kotlin-clean-architecture-1ad42fcd97fa).
```

Comment:
This image of MVVM architecture are wrong, because this picture shows mvvm arch using Repository pattern, try to use the right iamge

Reply:
done, thanks